java/opendcs/src/main/java/decodes/hdb/algo/SideInflowAlg.java

```diff
@@ -10,65 +10,57 @@
 import decodes.tsdb.VarFlags;
 // this new import was added by M. Bogner Aug 2012 for the 3.0 CP upgrade project
 import decodes.tsdb.algo.AWAlgoType;
+import org.opendcs.annotations.PropertySpec;
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
 
-//AW:IMPORTS
 import decodes.hdb.HdbFlags;
 
-//AW:IMPORTS_END
 
-//AW:JAVADOC
-/**
-This algorithm is an Side Inflow mass balance calculation for inflow as:  
-Delta Storage - Total Release Above + Total Release Below + evaporation 
+@Algorithm(description = "This algorithm is an Side Inflow mass balance calculation for inflow as:\n" +  
+"Delta Storage - Total Release Above + Total Release Below + evaporation\n\n" + 
 
-If inputs Delta Storage or Total Release Above or Total Release Below
-or  ithe Evap do not exist or have been deleted and the 
-Delta_STORAGE_MISSING or the TOTAL_REL_ABOVE_MISSING or EVAP_MISSING,
-or TOTAL_REL_BELOW_MISSING properties are set to "fail" then the 
-inflow will not be calculated and/or the inflow will be deleted.
+"If inputs Delta Storage or Total Release Above or Total Release Below\n" + 
+"or  ithe Evap do not exist or have been deleted and the\n" + 
+"Delta_STORAGE_MISSING or the TOTAL_REL_ABOVE_MISSING or EVAP_MISSING,\n" + 
+"or TOTAL_REL_BELOW_MISSING properties are set to \"fail\" then the\n" +  
+"inflow will not be calculated and/or the inflow will be deleted.\n\n" + 
 
-If all of the inputs do not exist because of a delete the inflow will 
-be deleted if the output exists regardless of the property settings.
+"If all of the inputs do not exist because of a delete the inflow will\n" +  
+"be deleted if the output exists regardless of the property settings.\n\n" + 
 
-This algorithm written by M. Bogner, August 2008
-Modified by M. Bogner May 2009 to add additional delete logic and version control
-
-
- */
-//AW:JAVADOC_END
+"This algorithm written by M. Bogner, August 2008\n" + 
+"Modified by M. Bogner May 2009 to add additional delete logic and version control")
```

Comment:
The comment about programmer,  should be a regular comment, not part of the annotation.